_posts/en/newsletters/2020-08-05-newsletter.md

```diff
@@ -0,0 +1,151 @@
+highlight a selection of the transcripts from the previous month.*
+
+FIXME:michaelfolkson
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding to update Tuesday
+
+- [C-Lightning 0.9.0][C-Lightning 0.9.0] is the newest major version of
+  C-Lightning.  It adds support for the updated `pay` command and new
+  `keysend` RPC described in [Newsletter #107][news107 notable].  It
+  also includes several other notable changes and multiple bug fixes.
+
+- [Bitcoin Core 0.20.1][Bitcoin Core 0.20.1] is a new maintenance
+  release.  In addition to bug fixes and some RPC behavior changes
+  resulting from those fixes, the planned release provides compatibility
+  with recent versions of [HWI][topic HWI] and its support for hardware
+  wallet firmware released for the [fee overpayment attack][].
+
+- [LND 0.11.0-beta.rc1][LND 0.11.0-beta] is the release candidate for a
+  new major release.  <!-- FIXME:harding to update Tuesday (LND haven't
+  yet published their draft release notes) -->
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19569][] allows Bitcoin Core to fetch the parents of _orphan_
+  transactions from peers that relay transactions using wtxid. An orphan
+  transaction is an unconfirmed transaction that we receive from a peer where we
+  don't yet have the parent transactions, either as part of of our best block
+  chain, or in the mempool. More precisely, an orphan transaction has at least
+  one transaction input whose assocociated output is not in the UTXO set or our
+  mempool's outpoint map.
+
+    When we receive an orphan transaction, we place it in a temporary data
+    structure called the orphan set. We then ask the peer that sent us the
+    orphan to also send us the parent transactions that we don't yet have. We can
+    do that because the orphan refers to the txids of the parent transactions. We
+    simply send a `GETDATA` message containing those txids to the peer to
+    request the parent transactions.
+
+    For [wtxid relay peers][news108 wtxid relay], transactions are announced
+    and requested using the _wtxid_ of the transaction, not the _txid_. However,
+    orphan transactions only refer to their parents' txids, not wtxids, so it's
+    not possible to request the parent transaction using wtxid. [PR
+    #18044][Bitcoin Core #18044] (which introduced wtxid relay peers and was
+    merged last week) did not permit fetching parent transactions from wtxid peers.
```

Comment:
perhaps set off "(which introduced wtxid relay peers and was merged last week)" with commas rather than parentheses

Reply:
fixed